src/api/app/components/request_decision_component.rb

```diff
@@ -28,13 +28,4 @@ def confirmation
   def show_add_submitter_as_maintainer_option?
     @action.type == 'submit' && !@action.creator_is_target_maintainer
   end
-
-  # TODO: Move all those "can_*" checks to a pundit policy
-  def can_accept_request?
-    @bs_request.state.in?(%i[new review]) && @is_target_maintainer
```

Comment:
Thanks to this line removed I think the `@is_target_maintainer` variable can be dropped, and the parameter passed on the Component initialization call too.

Reply:
Like this?

Reply:
If I'm not wrong, `@is_target_maintainer` is still being used on line 10 of `src/api/app/components/request_decision_component.html.haml`. See: https://github.com/openSUSE/open-build-service/pull/16958/files#diff-79f2aa701b809591100770e467ebdc17cd00bd45c6d900cf0c8f764694122562L10

Reply:
True, let me change it